recipes/GeoTessCPP/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -e
+
+make all CC=${CC}
```

Comment:
Is there a `make check` we can run here?

Reply:
No, there doesn't seem to be.